front_end_project/src/pages/resources/iotResources.js

```diff
@@ -1,102 +1,69 @@
-import { Container, Row, Col } from "react-bootstrap";
 import React, { Component } from "react";
-import "bootstrap/dist/css/bootstrap.css";
-import "bootstrap/dist/css/bootstrap.min.css";
 import tech from "./images/tech.png";
 import IoTMelbourne from "./images/IoTMelbourne.jpg";
 import device from "./images/device.jpg";
 
 class IotResources extends Component {
   render() {
     return (
-      <div
-        style={{
-          display: "flex",
-          flexDirection: "row",
-          flexWrap: "wrap",
-          justifyContent: "space-between",
-          padding: "20px",
-          maxWidth: "1500px",
-          margin: "auto",
-          backgroundColor: "#67A170"
-        }}
-      >
-        <Container>
-          <h1 class="text-center pt-3 pb-3">RESOURCES</h1>
-        </Container>
+      <div className="m-auto flex max-w-[1500px] flex-row flex-wrap justify-between bg-[#67A170] p-5">
+        <div className="2xl:max-w-[1320px] ml-auto mr-auto w-full sm:max-w-[540px] md:max-w-[720px] lg:max-w-[960px] xl:max-w-[1140px]">
+          <h1 className="pb-3 pt-3 text-center">RESOURCES</h1>
+        </div>
 
-        <Container
-          style={{
-            backgroundColor: "#D1E2C4",
-            maxWidth: 400,
-            marginBottom: "50px",
-          }}
-        >
-          <Col>
-            <Row>
+        <div className="mb-[50px] ml-auto mr-auto w-full max-w-[400px] bg-[#D1E2C4]">
```

Comment:
Since this component is reused thrice over the page. It would be a good idea to create a separate component and  use the image and the title as props. This is just so we ensure the code doesn't get repeated. DRY.